ee/fe/modules/500-openvpn/openapi/config-values.yaml

```diff
@@ -18,6 +18,7 @@ properties:
     type: object
     description: |
       A section of optional parameters of the `LoadBalancer` inlet.
+    default: { }
```

Comment:
Will work fine, but breaks consistency with `{}` elsewhere. Even in the same file: https://github.com/deckhouse/deckhouse/blob/e83377c1233c8bc7299cfce7e352e5805fe71856/ee/fe/modules/500-openvpn/openapi/config-values.yaml#L80